src/components/Icon/components/IconActionExpand.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const IconActionExpand = ({ viewBox = '0 0 28 28' }: { viewBox?: string }) => (
+  <svg viewBox={viewBox}>
+    <g fill="none" fillRule="evenodd">
+      <circle cx="14" cy="14" r="14" fill="currentColor" opacity=".5" />
+      <path
+        d="m6.858 16.286.572 4.285 4.291.572-.016.857H6v-5.714h.858Zm14.285-4.572L20.57 7.43l-4.291-.572L16.274 6H22v5.688l-.857.026Z"
+        fill="#fff"
```

Comment:
I considered `currentColor` here, but our default state for that icon is to have white arrows, and if `currentColor` used we would need to update it in our codebase.

With both `currentColor`:

![Screenshot 2022-12-09 at 12 08 08](https://user-images.githubusercontent.com/10700103/206700283-c1ffa92e-84bc-4f16-8b62-90c9ea21c7e7.png)